tests/Rector/ClassMethod/MakeEntitySetterNullabilityInSyncWithPropertyRector/Fixture/many_to_one_nullable.php.inc

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Rector\Doctrine\Tests\Rector\ClassMethod\MakeEntitySetterNullabilityInSyncWithPropertyRector\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ */
+class ManyToOneNullable
+{
+    /**
+     * @ORM\ManyToOne(targetEntity="NullableEntity")
+     * @ORM\JoinColumn(nullable=true)
+     */
+    private $nullableEntity;
+
+    public function setNullableEntity(NullableEntity $nullableEntity)
+    {
```

Comment:
Same as above the business logic can allow that something is nullable in the database but the setter is not.